src/parser.ts

```diff
@@ -1,14 +1,14 @@
-import { Flags } from '@oclif/core';
-import { Parser, convertToOldAPI } from '@asyncapi/parser/cjs';
 import { AvroSchemaParser } from '@asyncapi/avro-schema-parser';
 import { OpenAPISchemaParser } from '@asyncapi/openapi-schema-parser';
+import { Parser, convertToOldAPI } from '@asyncapi/parser/cjs';
 import { RamlDTSchemaParser } from '@asyncapi/raml-dt-schema-parser';
-import { getDiagnosticSeverity } from '@stoplight/spectral-core';
-import { html, json, junit, stylish, teamcity, text, pretty } from '@stoplight/spectral-cli/dist/formatters';
+import { Flags } from '@oclif/core';
 import { OutputFormat } from '@stoplight/spectral-cli/dist/services/config';
+import { getDiagnosticSeverity } from '@stoplight/spectral-core';
+import { html, json, junit, pretty, stylish, teamcity, text } from '@stoplight/spectral-formatters/dist';
```

Comment:
```suggestion
import { html, json, junit, pretty, stylish, teamcity, text } from '@stoplight/spectral-formatters';
```

I just changed this on local and tests started passing